### PR TITLE
Add caps for MP4 container buffers. Houskeeping.

### PR DESCRIPTION
**Pull request by mickel8**
For usage, refer to `ISOM` muxer in https://github.com/membraneframework/membrane_mp4_plugin/pull/39

### CODE REVIEW

File: lib/membrane_caps_mp4/mp4.ex [hunk 1]
```diff
@@ -0,0 +1,7 @@
+defmodule Membrane.MP4 do
```
Comment (daniel-jodlos) [inline lib/membrane_caps_mp4/mp4.ex:1]: I don't think this is actually used anywhere

File: lib/membrane_caps_mp4/mp4.ex [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  @moduledoc """
+  This module provides caps struct for MP4 container buffers.
```
Comment (daniel-jodlos) [inline lib/membrane_caps_mp4/mp4.ex:3]: These are not caps, this is an MP4 format if anything 😄 
```suggestion
  This module provides format definition for MP4.
```

File: lib/membrane_caps_mp4/mp4.ex [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+  """
+
+  defstruct []
```
Comment (mickel8) [inline lib/membrane_caps_mp4/mp4.ex:6]: Assuming we use this as `{RemoteStream, content_format: Membrane.MP4, type: :bytestream}`, is there the need to define struct or only module?

**Review by daniel-jodlos — CHANGES_REQUESTED** [review]

File: lib/membrane_caps_mp4/mp4.ex (hunk 3 above)
Comment (mat-hek) [reply lib/membrane_caps_mp4/mp4.ex:6]: Let's have the struct for consistency

**Review by mat-hek — APPROVED** [review]

**Review by daniel-jodlos — APPROVED** [review]

**Pull request merged**